terraform/api_gateway.tf.json.template.py

```diff
@@ -193,6 +193,55 @@ def for_domain(cls, domain):
                     'rule': [
                         {**rule, 'priority': i}
                         for i, rule in enumerate([
+                            ({
+                                 'name': 'HCAProdDownloadBudget',
+                                 'action': {
+                                     'block': {
+                                         'custom_response': {
+                                             'response_code': 429,
+                                             'response_header': [
+                                                 {
+                                                     'name': 'Retry-After',
+                                                     'value': str(config.waf_rate_rule_retry_after)
```

Comment:
Let's discuss this in PL.

Reply:
Agreed at PL to calculate a more accurate value for the header based on the observed number of concurrent downloads.

Reply:
Average number of distinct IPs requesting file downloads per 10 minute window in the last 4 weeks on HCA prod:

```
fields @message
| filter path like /^(fetch)?\/repository\/files/
| stats count_distinct(identity_sourceIp) as conc_clients by bin(10m)
| stats avg(conc_clients)
```

Result: 2.931

Reply:
...Coincidentally, this means that the ideal recommended backoff comes out to (60 * 10 / 59 * 2.9) = 29.5 seconds, almost exactly the existing retry value.